AISKU/package.json

```diff
@@ -65,12 +65,15 @@
     "dependencies": {
         "@microsoft/dynamicproto-js": "^1.1.4",
         "@microsoft/applicationinsights-shims": "2.0.0",
-        "@microsoft/applicationinsights-analytics-js": "2.6.5",
-        "@microsoft/applicationinsights-channel-js": "2.6.5",
-        "@microsoft/applicationinsights-common": "2.6.5",
-        "@microsoft/applicationinsights-core-js": "2.6.5",
-        "@microsoft/applicationinsights-dependencies-js": "2.6.5",
-        "@microsoft/applicationinsights-properties-js": "2.6.5"
+        "@microsoft/applicationinsights-analytics-js": "2.7.0-beta.1",
+        "@microsoft/applicationinsights-channel-js": "2.7.0-beta.1",
+        "@microsoft/applicationinsights-common": "2.7.0-beta.1",
+        "@microsoft/applicationinsights-core-js": "2.7.0-beta.1",
+        "@microsoft/applicationinsights-dependencies-js": "2.7.0-beta.1",
+        "@microsoft/applicationinsights-properties-js": "2.7.0-beta.1"
     },
-    "license": "MIT"
+    "license": "MIT",
+    "publishConfig": {
```

Comment:
This additional tag stops npm from make this as the "latest" it will be called "beta" instead